public/components/demo.js

```diff
@@ -0,0 +1,19 @@
+var video1Complete = false;
```

Comment:
Can you explain what problem this code is trying to solve?

Reply:
I need to check that both videos are done before the loop. I created the boolean to check and when there both true, the videos will restart. I had to do it this way, that way the loop can go on with infinity without the videos ever overlapping each other.

Reply:
I noticed that this can cause one video to not play at all (e.g  video1 only plays /after/ video2 has ended). 

Taking a step back, what is the underlying problem you're trying to solve? It seems like the videos are out of sync because they have slightly different lengths? If that's the case, could we fake it by just muting the audio for now and enhancing the context with some text?

Reply:
It works perfectly fine on my local copy. I think part of the issue is that if the wifi is bad or there's a lot of things being imported from links, the videos will load at different times and create the issue. From what I've tested, once in a while if you're opening the page for the **very** first time, it takes a little longer to render the images. However, after the **very** first time, ts runs smoothly.

Reply:
If you look at the JS again, there's an on end listener. So it only runs after the videos are done. I tried to make both videos the same length and there are still issues because they're like nanoseconds apart. JS seems to be the only option.

Reply:
Okay after reading the description of the problem in the issue this makes more sense to me. There still might be less technical solutions however (read: ones that don't involve code).

For example, what if you just captured the recordings so you just have one file to loop vs two. Get it to work/play once and then record that...this way you don't have any new js to write/maintain and you still "demo" the site functionality.

In general, try to think to yourself: "Do I _need_ to write code for this or is there some low/no code way to do it"

Reply:
Can you add a detailed comment of the problem here with a link to the bug?